logging/tests/unit/handlers/test__helpers.py

```diff
@@ -63,6 +64,57 @@ def test_valid_context_header(self):
         self.assertEqual(trace_id, expected_trace_id)
 
 
+# webapp2 has not been ported to python3 yet, so do not try to test its
+# functionality there.
+if sys.version_info[0] == 2:
+    class Test_get_trace_id_from_webapp2(unittest.TestCase):
+
+        @staticmethod
+        def create_app():
+            import webapp2
+
+            class GetTraceId(webapp2.RequestHandler):
```

Comment:
That would work, but it seems a bit ugly to have to repeat the exception-list here.  That said, if I go to unittest.skip it's probably going to be necessary.